### ISSUE TITLE
Keywords

### ISSUE THREAD

**Issue by zebdelk** [cedardevs/feedback#6]
On Monday 5/20 we had a meeting with metadata folks and one topic discussed was GCMD keywords vs other keyword dictionaries and types.

There are three main categories that keywords fall under:
- GCMD, which has been the priority for filtering and display in OneStop so far
- other keyword dictionaries, domain specific
- free text keywords

The primary goals:
- [ ] display all keywords, not just GCMD
- [ ] make it clear that existing keyword facets are specifically GCMD
- [ ] allow some type of filtering / facets based on non-GCMD keywords

Challenges and Tasks:
- [ ] Displaying all keywords on the collection details page should be fairly straightforward, and can have an additional category section next to the GCMD sections (which may need to be relabelled to identify them as GCMD keywords)
- [ ] Nicely formatting and organizing a wide variety of keywords with varying levels of formality and support may be a challenge. 
  - [ ] what benefits can we get from pulling in formal definitions of dictionaries? (eg. with RDF specs)
- [ ] One big concern is that dictionaries will have some overlapping terms with other dictionaries, and it may be difficult or infeasible to either 
  - search keywords regardless of dictionary source for overlaps in meaning / terms (eg. find everything marked with any dictionary concept of 'ocean')
  - organize keywords in such a way to make clear that there are multiple keywords with similar concepts that not all records will be marked with the same variation (eg. some records have 'Ocean' and others 'Oceans' would not be aggregated together)
- [ ] Is there a use for a free-form text box to search / autocomplete keywords? How difficult would something like that be?

In short, we can definitely expand support for other keywords, but we should be careful to balance user experience as much as we can.

**Comment by chris-noaa-affiliate** [comment]
Non-GCMD keywords also don't display in the OneStop record overview.  Here is an [example NMFS record](https://inport.nmfs.noaa.gov/inport-metadata/NOAA/NMFS/PIFSC/iso19115/xml/5568.xml).  And here is the [record in Test OneStop](https://data.test.ncdc.noaa.gov/onestop/collections/details/AW2jy-y65HQoqL5aoD8s) 


![Screen Shot 2019-10-06 at 7 29 27 PM](https://user-images.githubusercontent.com/29871707/66281756-ed36f700-e879-11e9-9bcb-8e2090224be9.png)
